lib/thrift/generator/server/binary_framed.ex

```diff
@@ -95,7 +95,7 @@ defmodule Thrift.Generator.Server.BinaryFramed do
       Thrift.TApplicationException ->
         quote do
           unhandled ->
-            Logger.error("Unhandled exception: #{Exception.message(unhandled)}")
+            Logger.error("Exception not defined in thrift spec was thrown: #{Exception.message(unhandled)}")
```

Comment:
Works for me.  Just something to clue the user in on where the exception came from.